src/updates.js

```diff
@@ -230,6 +219,7 @@ class Updates {
         if (rres.status < 400) {
           const body = await rres.text();
           const matches = body.match(/[^ ]*\.nupkg/gim);
+          assert(matches);
```

Comment:
Not sure if this is necessary but `ts-check` is telling me that `matches[0]` can be `null`.